core-services/prow/02_config/openshift-assisted/assisted-installer-deployment/_pluginconfig.yaml

```diff
@@ -1,3 +1,8 @@
+triggers:
```

Comment:
let's do this for all required repos, shall we?
* let's move this to ``core-services/prow/02_config/openshift-assisted/_pluginconfig.yaml``. this way it will apply for all the repositories in the ``openshift-assisted`` organization
* assisted-image-service misses this configuration as well